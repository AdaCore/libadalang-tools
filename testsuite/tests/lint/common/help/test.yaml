title: help

description: Tests the output of the --help flag

driver: shell_script
