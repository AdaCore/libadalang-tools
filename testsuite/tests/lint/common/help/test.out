usage: gnatrefactor(?:\.exe)? \[--help\|-h] \[--help\|-h] tool \[--verbose\|-v] \[--pipe\|-p]\s*
\s*\[--project\|-P PROJECT] \[--sources\|-S SOURCES \[SOURCES\.\.\.]]\s*
\s*\[-X nm=val \[nm=val\.\.\.]] \[--from-gnat-warnings\|-fgw\s*
\s*<file_with_gnat_warnings>]\s*

Refactor tools

positional arguments:
   tool  array_aggregates
   
optional arguments:
   --help, -h                  Show this help message
   --help, -h                  Help
   --verbose, -v               Print traces
   --pipe, -p                  Print the result to stdout instead of editing the
                                files on disk
   --project, -P               Project
   --sources, -S               Source files to refactor
   -X                          Specify an external reference for Project Files
   --from-gnat-warnings, -fgw  File with all obsolescent syntax GNAT warnings
   

